dashboard/test/models/script_test.rb

```diff
@@ -2035,13 +2035,11 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
     assert_equal 'Expect all lesson groups to have display names. The following lesson group does not have a display name: content1', raise.message
   end
 
-  test 'raises error if lesson group key already exists and try to change the display name' do
```

Comment:
This looks really weird. I deleted an old test and added a new tests. They are not really connected.